NGCHM/WebContent/javascript/PdfGenerator.js

```diff
@@ -1255,9 +1320,8 @@ PDF.pdfCancelButton = function() {
 	    //Get Dimensions for the Summary Heat Map
 	    const { sumMapW, sumMapH } = setSummaryHeatmapDimensions(sumImgW, sumImgH, rowTopItemsLength, colTopItemsLength);
 
-		const minDPI = 600;
-		const mapWidthScale = Math.max (2, Math.ceil (minDPI/72 * sumImgW / heatMap.getNumColumns (MAPREP.SUMMARY_LEVEL)));
-		const mapHeightScale = Math.max (2, Math.ceil (minDPI/72 * sumImgH / heatMap.getNumRows (MAPREP.SUMMARY_LEVEL)));
+		const mapWidthScale = Math.max (1, Math.ceil (pdfDoc.resolution/72 * sumImgW / heatMap.getNumColumns (MAPREP.SUMMARY_LEVEL)));
```

Comment:
Why did minimum "scale" change from 2 to 1 ?  And why magic 72 was ever used here?  I looked for anything that stated pdf resolutions were in points or that we kept our scales in points or something, but no?  It at least deserves a comment, but I haven't yet figured out what it's doing.

Reply:
I chose it because basically because it's the default number of points per inch in PDF documents.

But as I said in response to one of James' comments, I'm happy to change it to 100 (midway between the current 50 and 150 DPI entries).

Reply:
I guess that was actually _maximum_ scale that changed from 2 to 1.  You never answered this question.  It's fine if "that's what it should be".

Reply:
It **is** changing the minimum scale to 1.  1 is the smallest value that "works" and it generates the smallest PDF files.  It was changed to allow that for very small DPI settings.